cuml/forest_inference_demo.ipynb

```diff
@@ -0,0 +1,341 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Forest Inference Library\n",
+    "The forest inference library is used to load saved forest models of xgboost, lightgbm or protobuf and perform inference on them. It can be used to perform both classification and regression. This notebook shows how to use the Forest Inference library with xgboost and lightgbm models.\n",
+    "\n",
+    "The model accepts both numpy arrays and cuDF dataframes. In order to convert your dataset to cudf format please read the cudf documentation on https://rapidsai.github.io/projects/cudf/en/latest/. \n",
+    "\n",
+    "For additional information on the forest inference library please refer to the documentation on https://rapidsai.github.io/projects/cuml/en/latest/index.html"
```

Comment:
Great start. Let's expand a little bit to give a tour of this notebook so people don't get confused why there is fitting there too. Maybe add a little more like:

Although FIL is focused on inference, in this notebook, we'll begin by fitting a model with XGBoost and saving it. We'll then load the saved model into FIL and use it to infer on new data.

Reply:
I do agree with John, but also want to point out (unrelated) that I really love the links to the docs. This is such a good design and will make these notebooks more maintainable going forward.